src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  grid-template-columns: repeat(4, 200px);
+  grid-auto-flow: row;
+
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 468px) {
+    grid-template-columns: repeat(1, 200px);
```

Comment:
Must be one column below 4**88**. you did one column below 4**68** **inclusive**.  You got the number wrong

Reply:
Oh, really! Thank you